src/v/kafka/server/request_context.h

```diff
@@ -183,7 +183,7 @@ class request_context {
         }
 
         vlog(
-          klog.trace,
+          klog.info,
```

Comment:
Won't this blow up the logs at the info level? IE will this record all successful produce/fetch requests too?

Should we only log errors at info and only errors for specific response types?

Reply:
>Won't this blow up the logs at the info level? IE will this record all successful produce/fetch requests too?

yeh, i don't think we can make this exact change to info level. it will generate far too much logging output.

Reply:
From the slack messages, it seemed the desire was for INFO logs when they were searching for the `NOT_COORDINATOR` error. Adding an INFO/ERROR log for specific response types may be the path forward here.

Reply:
After discussion in slack, the decision was to switch from TRACE to DEBUG.

Reply:
IIRC the decision was to log **errors only** at DEBUG, and sucessful requests at TRACE.